lib/ansible/modules/storage/netapp/na_ontap_net_vlan.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: na_ontap_net_vlan
+short_description: Manage NetApp Ontap network vlan
+extends_documentation_fragment:
+    - netapp.ontap
+version_added: '2.6'
+author: Chris Archibald (carchi@netapp.com), Kevin Hutton (khutton@netapp.com)
+description:
+- Create or Delete a network vlan
+options:
+  state:
+    description:
+    - Whether you want to create to delete a network vlan
+    choices: ['present', 'absent']
+    default: present
+  parent_interface:
+    description:
+    - The interface that hosts the vlan interface.
+    required: true
+  vlanid:
+    description:
+    - The vlan id. Ranges from 1 to 4094.
+    required: true
+  node:
+    description:
+    - Node name of vlan interface.
+  interface_name:
+    description:
+    - Name of vlan interface. The name must be of the format <parent-inteface>-<vlanid>
+  gvrp_enabled:
```

Comment:
`type: bool`? `default: false` (and remove the redundant default text)

Reply:
Fixed.